playstore/notification.go

```diff
@@ -38,6 +38,7 @@ const (
 // DeveloperNotification is sent by a Pub/Sub topic.
 // Detailed description is following.
 // https://developer.android.com/google/play/billing/rtdn-reference#json_specification
+// Deprecated
```

Comment:
Deprecated comments should be `Deprecated: `.
For example, `Depreacated: use DeveloperNotificationV2 intead`. 
It would be kind to do it this way ☺️ 
https://go.dev/wiki/Deprecated

Reply:
ok

Reply:
`Depreacated: use DeveloperNotificationV2 intead.`
Please write on one line 🙏